app/mailers/shf_application_mailer.rb

```diff
@@ -9,7 +9,7 @@ def acknowledge_received(shf_application)
 
 
   def app_approved(shf_application)
-    @branding_fee_paid = shf_application.company&.branding_license?
+    @branding_fee_paid = shf_application.companies.last&.branding_license?
```

Comment:
I did not change this because:

1. We currently only support the ability of a user to associate one company with  an application (in the UI).  So, we _shouldn't_ see a situation involving more than one company until we relax that constraint.  Whether supporting this case would involve a re-review of the application is not decided.

2. _That_ situation might arise when we figure out how to handle application renewals and/or adding a category to an existing application (at which point it is very possible to have more than one company associated with the app).  But, we don't support that now either.

My thinking on this is that this email would probably go out _once_ to the user - when their application is first accepted.  

Case #1 above would probably require another email type altogether - that is, we're not reviewing the application but simply associating that app with another company (and, if the member can do this themselves, no email is required).

Case #2 would almost certainly require a different email (or emails - one for bi-annual review completion and one for adding one or more categories).

So, I had the same thoughts as you when I adjusted this code.  Given my thoughts above, however, I decided to leave it alone for now - as we think through these refactoring side-effects which are very fundamental to how we model the SHF world.

I also think that the best way to deal with this is to provide this email action with the specific company involved with the initial application, so it doesn't have to perform this seemingly crazy logic.

I'll create a story for that if you agree?  Thoughts?